Snake.js

```diff
@@ -152,19 +152,16 @@ Snake.prototype.createGeometry = function( geometry  , numOf ){
   // Get the totalVerts by looking up how many faces
   // we've got in the geometry
 
-  var totalVerts = faces * 3 * numOf;
-
-  var positions = new Float32Array( totalVerts * 3 );
-  var lookups   = new Float32Array( totalVerts * 2 );
-  var normals   = new Float32Array( totalVerts * 3 );
```

Comment:
* Positions and normals occur once; rather than being repeated numOf times
* Lookups occur numOf times; rather than numOf * 3 * faces